src/styles/product-button.css

```
@@ -0,0 +1,21 @@
+.product-button {
+  box-sizing: border-box;
+  display: inline-block;
+  height: 40px;
+  width: 166px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  line-height: 38px;
```

не коригуй розміщення тексту за допомогою висоти лінії 
```suggestion
  text-align: center;
```